lib/maths/CTimeSeriesSegmentation.cc

```diff
@@ -847,38 +847,38 @@ void CTimeSeriesSegmentation::fitTopDownPiecewiseTimeShifted(ITR begin,
 
         // The following loop is the interesting part. We compute
         //
-        //   min_j{ min_s{ sum_{i < j}{(x_i - p_i(s))^2} } + min_s{ sum_{i >= j}{(x_i - p_i(s))^2} } }
+        //   min_i{ min_{sl, sr}{ sum_{j<i}{(x_j - p_{j+sl})^2} + sum_{j>=i}{(x_j - p_{j+sr})^2} } }
```

Comment:
Should the minimization problem here and above be the same?

Reply:
There was a typo in the above, i.e. it should have been x_j not x_ij. The i does really just appear in the summation limits, i.e. it selects where we split the data. Other than that the difference is how `sl` and `sr` act. In the above, it is a constant scale applied to the prediction. Here, it is a constant time shift applied to the argument, i.e. we're looking up at a different offset.